src/components/Events/EventCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
```

Comment:
@arunimaChintu Why have you added this file? Please explain.

Reply:
It was to show the event cards... it was just for demo, I will remove this...